### PR TITLE
docs: mention ci/CreateArchive_SkyrimPlatform.cmd in readme

### LINKED ISSUE #547: Mention ci/CreateArchive_SkyrimPlatform.cmd in docs

### PR DESCRIPTION
**Pull request by Pospelove**
closes #547

### CODE REVIEW

**Comment by Pospelove** [comment]
@nic11 review plz

File: README.md [hunk 1]
```diff
@@ -162,6 +162,14 @@ directory and add some aliases to `PATH`.
    ```
    These commands would re-generate project files with coverage enabled and run tests. Coverage report would be in `build/__coverage`.
 
+3. Create SkyrimPlatform archive with structure identical to our Nexus uploads (Windows-only):
```
Comment (nic11) [inline README.md:165]: I know it's located inside of optional steps section, but I think that it should be clarified somehow of why one could want to to this. Something like 'If you wish to use solely SP build, you can run this script to get zip with only related files'

And a few thoughts more (out of scope of this PR):
1. Always packing files look unnecessary, I guess a frequent case would be 'build -> create zip from **some of** dist files -> unpack zip -> use it in MO2', so...
2. ... probably it makes more sense to make a script that copies files to a separate folder instead (or just deletes extra files in place)
3. Or, ideally, having an option to run some target that will only add those files to `build/dist` and nothing more

File: README.md [hunk 2]
```diff
@@ -162,6 +162,14 @@ directory and add some aliases to `PATH`.
...
+   
+   From the repo root:
+   ```sh
+   cd ci
+   ./CreateArchive_SkyrimPlatform.cmd
```
[the remark this answers is not in the document]
Comment (nic11) [reply README.md:170]: And well I guess it'll be simpler to just say 'double click that .bat file' or run it from console

**Comment by Pospelove** [comment]
@nic11 Thanks for your feedback! I think I still need to rework this PR, even if some items may be considered as out of scope.

**Comment by Pospelove** [comment]
Fixed description, but kept unfixed out-of-scope items. `ci` seems to bee commonly used in our project at the moment of writing this comment. So I decided to go forward with this and merge..

**Pull request merged**